src/components/dragBoxLayer.ts

```diff
@@ -106,6 +118,14 @@ export module Components {
           } else if (resizingEdges.left) {
             topLeft.x = e.x;
           }
+        } else {
```

Comment:
Can we do a switch case? It's easier to figure out what this else corresponds to